portability-api/src/main/java/org/dataportabilityproject/webapp/PortabilityApiMain.java

```diff
@@ -52,6 +55,8 @@ public static void main(String args[]) throws Exception {
     portabilityJobFactory = new PortabilityJobFactory(new UUIDProvider());
     // TODO: Wire up the correct Crypter.
     cryptoHelper = new CryptoHelper(jobDao);
+    // TODO: Wire up correct secret key
+    tokenManager = new JWTTokenManager("supersecretapistring");
```

Comment:
will this be stored in our key management system?

Reply:
Yes, this should be stored in the Cloud KMS and not checked into the repository.